### ISSUE TITLE
More documentation for tools

### ISSUE THREAD

**Issue by Hume2** [SuperTux/supertux#337]
Actually, the tools we have in tools/ have just a little documentation and not much useful. It should be there described well what purpose does it serve and what's the input and output. There's a summary of what is missing:
*autopackage: No documentation at all
*darwin: There's a picture, but it's unclear whether there's something missing or it demonstrates the result. There's absolutely no written text in it.
*innosetup: It makes innosetup, but input/output?
*miniswig: If I didn't know that it's for wrapper.cpp, I'd have no idea what does it serve for.
*build-addon-index: Good, but input/output?

There are also some tools in the tools/ directory, not sorted out. I think, it needs to sort them out to separate dirrectories.

**Comment by maxteufel** [comment]
Agreed; this should be in the wiki once we're done cleaning up the imported files.

> - build-addon-index: Good, but input/output?

This one should be moved to the addons repository.

**Comment by maxteufel** [comment]
> - darwin: There's a picture, but it's unclear whether there's something missing or it demonstrates the result. There's absolutely no written text in it.

This one is pretty obvious for OS X users (which uses the Darwin kernel) - it's actually a background for the DMG - It's supposed to tell the user to move the SuperTux.app to the Applications folder.

**Comment by Karkus476** [comment]
@maxteufel Why not include it in the [doc directory](https://github.com/SuperTux/supertux/tree/master/docs) too, for offline reference?

**Comment by maxteufel** [comment]
> @maxteufel Why not include it in the doc directory too, for offline reference?

@Karkus476: Might be a good idea for specific documentation on core features, however, we can't put a copy of the entire wiki in there and keep it up to date.
